src/Client.cpp

```diff
@@ -218,6 +218,15 @@ void Client::OnAfterCreated(CefRefPtr<CefBrowser> browser)
 
     CEF_REQUIRE_UI_THREAD();
 
+    CefRefPtr<CefFrame> frame = browser->GetMainFrame();
+
+    frame->ExecuteJavaScript(
+        "window.triggerCefPdf = function () { window."
```

Comment:
We can make it even better; `window.cefpdf.print()`. Make an object `cefpdf` with methods `print` and `cancel` or `abort`. It can be extended in the future.